src/Bridge.ts

```diff
@@ -231,11 +231,11 @@ export class Bridge {
             this.github?.onInstallationRemoved(data.data);
         });
 
-        this.bindHandlerToQueue<GitHubWebhookTypes.IssueCommentCreatedEvent, GitHubIssueConnection>(
+        this.bindHandlerToQueue<GitHubWebhookTypes.IssueCommentCreatedEvent, GitHubIssueConnection|GitHubRepoConnection>(
             "github.issue_comment.created",
             (data) => {
                 const { repository, issue, owner } = validateRepoIssue(data);
-                return connManager.getConnectionsForGithubIssue(owner, repository.name, issue.number).filter(c => c instanceof GitHubIssueConnection) as GitHubIssueConnection[];
+                return connManager.getConnectionsForGithubIssue(owner, repository.name, issue.number);
```

Comment:
The other one works in GitHub issue rooms as oppose to repo rooms, and it's quite a bit different in that it uses actual puppet users rather than a single bot user.